samples/excel/excel-library/operations-on-workbooks/BlazorClientApp.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <RazorLangVersion>3.0</RazorLangVersion>
+    <AssemblyName>Infragistics.Samples</AssemblyName>
+    <RootNamespace>Infragistics.Samples</RootNamespace>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>1701;1702,IDE0028,BL0005,0219</NoWarn>
+  </PropertyGroup>
+    
+  <ItemGroup>
+      <PackageReference Include="IgniteUI.Blazor" Version="20.2.26" />
+      <PackageReference Include="IgniteUI.Blazor.Documents.Excel" Version="20.2.33" />
```

Comment:
why these 2 IG assemblies do not have the same version? Also, check if other excel samples have the same issue